cf_units/tests/integration/test__num2date_to_nearest_second.py

```diff
@@ -342,4 +342,4 @@ def test_fractional_second_365_day(self):
 
 
 if __name__ == "__main__":
-    unittest.main()
+    pytest.main([__file__])
```

Comment:
This retains the ability to run individual test modules as

`python test_stuff.py`

May now be redundant as you can just point `pytest` at a single test module.  I note that neither Matplotlib nor Cartopy have `ifmain`s in their test modules.

Reply:
I would say we don't really have a need for this style anymore.
PytTest code does not really follow "normal" Python code idioms, due to its extensive use of inspection and patching. 
(even unittest is a bit like that, in discovery and setup/teardown)
So I think in adopting it we can simply accept that you will _always_ run it via pytest.

Reply:
Now removed https://github.com/SciTools/cf-units/pull/245/commits/d0bb08947dd6c919f796cde8ca89a1346fd0ea72